warehouse/packaging/models.py

```diff
@@ -415,6 +415,45 @@ class ReleaseURL(db.Model):
     url = Column(Text, nullable=False)
 
 
+class Dynamic(db.Model):
```

Comment:
I don't think we need this to be a relation. I'm going to push a change to convert it to an Array containing the Enum